Documentation/gettingstarted/ipam-crd.rst

```diff
@@ -79,14 +80,19 @@ Create a CiliumNode CR
          [...]
        spec:
          ipam:
-           192.168.1.1: {}
-           192.168.1.2: {}
-           192.168.1.3: {}
-           192.168.1.4: {}
+           pool:
+             192.168.1.1: {}
+             192.168.1.2: {}
+             192.168.1.3: {}
+             192.168.1.4: {}
        status:
          ipam:
            used:
              192.168.1.1:
                owner: router
              192.168.1.3:
                owner: health
+
+.. note::
+
+At the moment only single ip addresses are allowed. Cidr's are not supported.
```

Comment:
I think you missed the tab here :sweat_smile: 
Also,
ip -> IP
cidr -> CIDR 

```
.. note::

    At the moment only single IP addresses are allowed. CIDR's are not supported.
```

Similar to how note is formatted here - https://raw.githubusercontent.com/cilium/cilium/master/Documentation/beta.rst